### ISSUE TITLE
[HOLD for payment 2024-03-20] [$500] [Wave Collect] [More Features] Create WorkspaceMoreFeaturesPage in App

### ISSUE THREAD

**Issue by mountiny** [Expensify/App#37373]
Part of https://github.com/Expensify/Expensify/issues/365143

Implement [this section](https://docs.google.com/document/d/1xk-fp5tKM-Es5oBrlWa0oHFUfBJH-XaFWzywzCkRaEY/edit#bookmark=id.pdagmy7dygy) of the design doc. 

This is not on HOLD per se, we can develop the UI now and then merge once the backend is ready for it.

cc @narefyev91 
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0199f8f951b5430173</li>
        <li>Upwork Job ID: 1762624586850594816</li>
        <li>Last Price Increase: 2024-02-27</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @kevinksullivan</details>

**Comment by mountiny** [comment]
Not looking for external help, agency will handle it

**Comment by rezkiy37** [comment]
Hi, I’m Michael (Mykhailo) from Callstack and I would like to work on this issue.

**Comment by rezkiy37** [comment]
I created some draft UI. I need to clarify icons, titles, subtitles and so on. Please provide me more info. Thank you!

<img width="1728" alt="List" src="https://github.com/Expensify/App/assets/57314631/997cb4d5-b93b-4c7c-bca6-695fef4c3e44">

<img width="1728" alt="Empty" src="https://github.com/Expensify/App/assets/57314631/9b48b75a-d5e7-4348-a10d-594fea6adb8d">

**Comment by mountiny** [comment]
@shawnborton could you please help with https://github.com/Expensify/App/pull/37441#issuecomment-1969883125 thank you

**Comment by shawnborton** [comment]
Will do! I'll try to get to this tomorrow morning and if not, will reassign to another @Expensify/design member

**Comment by shawnborton** [comment]
Added them [here](https://github.com/Expensify/App/pull/37441#issuecomment-1971098947)

**Comment by rezkiy37** [comment]
Actively working on the issue.

**Comment by s77rt** [comment]
Not overdue. @mountiny Can you please assign @rezkiy37

**Comment by s77rt** [comment]
Not overdue. PR is in draft status

**Comment by rezkiy37** [comment]
Yes, opening the [PR](https://github.com/Expensify/App/pull/37441) for review today.

**Comment by trjExpensify** [comment]
Wahoo! @s77rt are you good to jump on this review today, please? Thanks!

**Comment by s77rt** [comment]
Sure! Will start reviewing asap.

**Comment by trjExpensify** [comment]
Perfect, thanks!

**Comment by s77rt** [comment]
**Regression Test Proposal**

(Requires a paid workspace with admin role)

1. Open App
2. Go to Profile > Workspaces > Paid workspace
3. Verify there is a tab "More features" and click on it
4. Enable any feature
5. Verify the feature is added to the LHP
6. (Web/Desktop only) Verify you are redirected to that feature's page
7. Disable any feature
8. Verify the feature is removed from the LHP

**Comment by luacmartins** [comment]
We'll add regression tests as part of the design doc once the feature is fully implemented.

**Comment by shawnborton** [comment]
Not overdue, looks like we're working on payment details above.

**Comment by mountiny** [comment]
$500 to @s77rt

**Comment by luacmartins** [comment]
@kevinksullivan Was this paid out? Can we close it?

**Comment by luacmartins** [comment]
Bump for payment @kevinksullivan

**Comment by luacmartins** [comment]
Bump for payment

**Comment by s77rt** [comment]
Payment due is $125 for two regressions (https://github.com/Expensify/App/issues/38185 and https://github.com/Expensify/App/issues/38198)

**Comment by kevinksullivan** [comment]
sorry for the delay, new job created and paid.
